lib/simple_lru.js

```diff
@@ -43,10 +44,12 @@ Cache.prototype.get = function(key,hit){
      * it to the head of linked list  
      */
     hit = hit != undefined && hit != null ? hit : true;
-    if(cacheVal && hit)
+
+    if(cacheVal && hit && (!cacheVal.time || (Date.now() - cacheVal.time) < this.maxTime)) 
```

Comment:
You'll need to `delete this.cache[key]` if it's expired to keep the cache clean.